src/JiraCli/JiraCli.csproj

```diff
@@ -71,6 +71,10 @@
       <HintPath>..\..\lib\RestSharp.105.1.0\lib\net4\RestSharp.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Semver">
+      <HintPath>..\..\lib\semver.1.1.2\lib\net40\Semver.dll</HintPath>
```

Comment:
I think we already have a SemVer model? If we use a lib, we need to make sure that we embed it using Costura.